pkg/server/controllers.go

```diff
@@ -693,6 +696,55 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 	})
 }
 
+func (s *Server) installAPIBinderController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	config = rest.CopyConfig(config)
+	config = rest.AddUserAgent(config, initialization.ControllerName)
+
+	// Client used to create APIBindings within the initializing workspace
+	initializingWorkspacesCfg := config
+	initializingWorkspacesCfg.Host = fmt.Sprintf("https://%v/services/%v/%v", s.GenericConfig.ExternalAddress, initializingworkspaces.VirtualWorkspaceName, tenancyv1alpha1.ClusterWorkspaceAPIBindingsInitializer)
+	initializingWorkspacesKcpClusterClient, err := kcpclient.NewClusterForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	// TODO: put in server.go or config.go.
+	initializingWorkspacesKcpInformers := kcpexternalversions.NewSharedInformerFactory(
```

Comment:
I tried moving this into config.go as a shared controller, but would get into a loop where the virtual workspace wasn't ready yet. I'm not exactly sure where this should land.